platform/README.md

```diff
@@ -4,39 +4,15 @@
 
 ## Running code
 
-To get this stuff running locally (as we don't have a proper build system yet):
-
-From a clean setup locally (the best is probably to just `rm` your
-`node_modules`), run:
-
 ```sh
-#  in root directory
-yarn
-
-cd packages/kbn-internal-native-observable
-yarn
-cd -
-
-cd packages/kbn-observable
-yarn
-cd -
-
-cd packages/kbn-utils
-yarn
-cd -
-
-cd packages/kbn-types
-yarn
-cd -
-
-# in root directory
+# all commands from the root directory
+yarn kbn bootstrap
+yarn kbn run build --skip-kibana --skip-kibana-extra
```

Comment:
Not super-nice yet, but I've got some ideas I'm planning on exploring early next week